src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -245,33 +249,55 @@ You should see:
 
 Our `nri-bundle` chart. whose installation instructions can be found above, acts as a wrapper or a meta-package for a couple of other charts, which are the ones containing the components for our solution. By offering such a wrapper we can provide a controlled set of our components with versions that we know are compatible with each other, while keeping the component's charts relatively simple.
 
-To configure the individual integration components, you must use Helm's dependency system, which simply means that configuration for the children charts must be put under a section with the name of said chart. For example, to configure the `newrelic-infrastructure` chart, you would add the following to the `values-newrelic.yaml`:
+The [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart wraps [multiple individual charts](#install-k8-helm) to gather different telemetry data and send it to New Relic. The bundle allows to selectively enable the desired child charts depending on your needs. To configure each individual component, you must use [Helm's dependency system](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart), which in short means that the configuration for each children chart must be placed under a separate section (named after each child chart) in the [values-newrelic.yml file](https://helm.sh/docs/chart_template_guide/values_files/). For example, to configure the `newrelic-infrastructure` chart, you would add the following to the `values-newrelic.yaml`:
 
 ```yaml
+# General settings that apply to all the children charts
```

Comment:
children -> child (I _think_ 😄), and same for the five usages below this.
 
You: "But 'charts' is plural, so I'm pluralizing the adjective, Brian!"
Me: "I know, I know, but... English. 👎🤷😄"

Reply:
Thanks!